plugins/georeplication/api/resp.go

```diff
@@ -0,0 +1,56 @@
+package georeplication
+
+import (
+	"github.com/pborman/uuid"
+)
+
+const (
```

Comment:
Just a FYI: These can be like enums instead. There's [stringer](https://godoc.org/golang.org/x/tools/cmd/stringer) tool which can make it strings. I got to know about this while reviewing the volgen PR. You can take a look at it if it makes things simpler/easier for you.

Reply:
Let me explore that, thanks. Is that libray already part of glusterd2?

Reply:
> Is that libray already part of glusterd2?

It's part of standard go tools and isn't an importable library. The volgen PR has example on how to use it. The stringer tool will generate code. All you have to do it include a one line comment in the go source file where you have your types (enum-like consts) defined. Example:

```go
//go:generate stringer -type=GeorepStatus
```

Reply:
I will include this in another patch later.